lib/npg_pipeline/function/seq_alignment.pm

```diff
@@ -33,6 +33,9 @@ Readonly::Scalar my $DEFAULT_SJDB_OVERHANG        => q{74};
 Readonly::Scalar my $REFERENCE_ARRAY_ANALYSIS_IDX => q{3};
 Readonly::Scalar my $REFERENCE_ARRAY_TVERSION_IDX => q{2};
 Readonly::Scalar my $DEFAULT_RNA_ANALYSIS         => q{tophat2};
+Readonly::Hash   my %RNA_ANALYSES                 => (q{tophat2} => 1,
+                                                      q{star}    => 1,
+                                                      q{salmon}  => 1);
```

Comment:
This really is menat to be a list. 'any' from List::MoreUtils can be used to search the list